### ISSUE TITLE
Could not load type 'Sitecore.Jobs.Job' SPE 5.1 Sitecore 9.2

### ISSUE THREAD

**Issue by Antonytm** [SitecorePowerShell/Console#1188]
### Expected Behavior

SPE 5.1 should work with Sitecore 9.2

### Actual Behavior

SPE throw exception on some of commands
```
Publish-Item : Could not load type 'Sitecore.Jobs.Job' from assembly 'Sitecore.Kernel, Version=13.0.0.0, Culture=neutral, PublicKeyToken=null'.
At line:1 char:91
+ ... r Arabia\Air Arabia\Air Arabia" | Publish-Item -Recurse -RepublishAll
+                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Publish-Item], TypeLoadException
    + FullyQualifiedErrorId : System.TypeLoadException,Cognifide.PowerShell.Commandlets.Data.PublishItemCommand
```

### Steps to Reproduce the Problem

1. Open Powershell ISE
2. Run command Get-Item -Path "master:\sitecore\media library\Themes\my theme" | Publish-Item -Recurse -RepublishAll

Sitecore version 9.2
SPE version 5.1
SXA 1.9

I have reviewed Stackoverflow questions and follow suggestion to use 5.1 version. It helped, SPE started to work almost for all queries, but not for all...  
https://sitecore.stackexchange.com/questions/20083/sitecore-powershell-extensions-module-compatibility-with-sitecore-9-2?noredirect=1&lq=1
https://sitecore.stackexchange.com/questions/20778/error-in-sitecore-9-2-and-sxa-1-9-could-not-load-type-sitecore-jobs-job-when?noredirect=1&lq=1

 - [X] Reviewed questions and answers on the Sitecore Stack Exchange.

**Comment by Antonytm** [comment]
I know, that there is other option to use Sitecore 9.2 + special version of SPE 5.0. 
But I have different problem with SPE 5.0. Remoting doesn't work, it throws 404 not found.

```
VERBOSE: Preparing to invoke the script against the service at url https://host/-/script/script/?user=Admin&password=b&sessionId=19486c2f-ca2f-481c-a3ef-90c0d0
c060ae&rawOutput=False&persistentSession=False
VERBOSE: Transferring script to server
VERBOSE: Response exception message: Exception calling "Dispose" with "0" argument(s): "The remote server returned an error: (404) Not Found."
VERBOSE: Response status description: 
VERBOSE: No response returned by the service. If results were expected confirm that the service is enabled and the account has access.
```

**Comment by michaellwest** [comment]
Related to #1129. Issue was fixed in 6.0. We discovered it after releasing 5.1.

**Comment by michaellwest** [comment]
I'm going to go ahead and close this issue. If you see it happen on 6.0+ then please reopen.
